### ISSUE TITLE
how to Update/Upgrade multiple agents using Dev Tools?

### ISSUE THREAD

**Issue by winfield27** [wazuh/wazuh#7987]

**Comment by winfield27** [comment]
PUT /agents/group/upgrade - is not working and prompting error 
{
  "error": "3013",
  "message": {
    "title": "Not Found",
    "detail": "404: Not Found"
  }
}

**Comment by CarlosRS9** [comment]
Hi @winfield27

That error means the endpoint you are trying to use to upgrade a Wazuh agent (`/agents/group/upgrade`) does not exists. If you want to upgrade several agents at the same time you can use the following endpoint:
```
PUT /agents/upgrade?agents_list=001,002,003
```
Replace `001,002,003` with the ids of the agents you want to upgrade. More information about this endpoint [here](https://documentation.wazuh.com/current/user-manual/api/reference.html#operation/api.controllers.agent_controller.put_upgrade_agents).

Take into account this endpoints is available for Wazuh 4.0.0 or later only. If you are using an old version, such as Wazuh 3.13.2 or lower you have to upgrade the agents one at a time using this endpoint instead:
```
PUT /agents/{agent_id}/upgrade
```
Replace `{agent_id}` with the id of the agent you want to upgrade. For example: `PUT /agents/001/upgrade`

Let me know if you have any question regarding this topic as I will try to address them.

Regards.

**Comment by CarlosRS9** [comment]
@winfield27 let me close this issue. Do not hesitate to reopen it in case of further questions or suggestions related to this topic.

Regards.
